kml/serdes_common.cpp

```diff
@@ -6,23 +6,28 @@
 namespace kml
 {
 
-std::string PointToString(m2::PointD const & org)
+std::string PointToString(m2::PointD const & org, char const * separator)
 {
   double const lon = mercator::XToLon(org.x);
   double const lat = mercator::YToLat(org.y);
 
   std::ostringstream ss;
   ss.precision(8);
 
-  ss << lon << "," << lat;
+  ss << lon << separator << lat;
   return ss.str();
 }
 
-std::string PointToString(geometry::PointWithAltitude const & pt)
+std::string PointToString(geometry::PointWithAltitude const & pt, char const * separator)
```

Comment:
Making the separator a `char const` (a single character) will speed up the code a bit.